inst/WORDLIST

```diff
@@ -1,86 +1,33 @@
+ADXX
 ADaM
-ADAE
-ADCM
-ADDV
-ADEC
-ADEG
-ADEX
-ADLB
-ADMH
-ADSL
-ADVS
-AE
-analytics
-ATC
-Bazett
-Bazett's
-BDS
-Biologics
-BMI
 Biologics
 CDISC
-Changelog
-censorings
-codebase
-CRF
-CQ
-cyclomatic
-datepart
-datetime
-developers’
-dtc
-DTC
-DuBois
-durations
-EMA
-FACM
-Fridericia
-Fridericia's
-Fujimoto
-functions’
-funder
-Gehan
-GitHub
-GlaxoSmithKline
-groupwise
+CMD
+Chagnelog
```

Comment:
I would think this is something we want flagged?

Reply:
agree - thats a typo somewhere!

Reply:
Do you mind fixing this in the News.md file and removing this from the wordlist?